storage/src/main/java/tech/pegasys/teku/storage/client/CombinedChainDataClient.java

```diff
@@ -563,6 +566,10 @@ public SafeFuture<Optional<UInt64>> getEarliestAvailableBlobSidecarSlot() {
     return historicalChainData.getEarliestAvailableBlobSidecarSlot();
   }
 
+  public SafeFuture<Optional<UInt64>> getEarliestAvailableDataSlot() {
```

Comment:
suggestion: call this one `getEarliestAvailableDataColumnSlot` because it literally calls variable with this name
+ call another one `getEarliestAvailableDataColumnSlotWithFallback`
we can discuss it separately